src/app/static/src/app/store/load/actions.ts

```diff
@@ -41,14 +43,14 @@ export interface modelOutputPayload {
 
 export const actions: ActionTree<LoadState, RootState> & LoadActions = {
 
-    async load({dispatch}, payload) {
+    async load(context, payload) {
+        const {dispatch} = context
         const {file, projectName, source} = payload;
 
         if (FileSource.ModelOutput === source) {
             const formData = new FormData()
             formData.append("file", file)
-            await dispatch("prepareModelOutput", {file: formData, projectName});
-
+            await dispatch("preparingRehydrate", {file: formData, projectName});
```

Comment:
This action dispatch still seems unnecessary to me. Not a showstopper but I don't think we should get into this habit, of spawning actions from actions instead of just calling methods, .

Reply:
I tried moving but introduced a different issue but don't worry II will sort this out in the UI ticket.